tools/get_release_contributors.py

```diff
@@ -28,7 +35,7 @@ class UnexpectedGithubStatus(RuntimeError):
 
 
 # List of user names which should be ignored (such as bots):
-ignore_list = ['github-actions[bot]', 'imgbot[bot]']
+ignore_list = ['github-actions[bot]', 'imgbot[bot]', 'actions-bot', 'ImgBotApp']
```

Comment:
```suggestion
ignore_list = ['github-actions[bot]', 'imgbot[bot]', 'actions-bot', 'ImgBotApp', 'actions-user']
```

As mentioned by @softins

Reply:
Thanks @softins and @ann0see. Added.